lib/plugins.js

```diff
@@ -830,7 +822,7 @@ exports.categories = {
             description: 'A configuration document format, an API, and a foundation for A/B testing'
         },
         faketoe: {
-            url: 'https://github.com/hapijs/faketoe',
+            url: 'https://github.com/drummel/faketoe',
```

Comment:
Not sure that's a good idea to add a link to that repo. `faketoe` package on `npm` is still owned by Eran etc. : https://www.npmjs.com/package/faketoe. Plus the person who forked the repo didn't add a link to another package and doesn't seem either to maintain it so I would just go and delete it from the list like the rest you deleted.

Reply:
i have deleted faketoe like the rest. Please review changes